impeller/renderer/backend/vulkan/context_vk.h

```diff
@@ -160,6 +179,8 @@ class ContextVK final : public Context,
   std::shared_ptr<FenceWaiterVK> fence_waiter_;
   std::string device_name_;
   std::shared_ptr<fml::ConcurrentTaskRunner> worker_task_runner_;
+  std::shared_ptr<fml::ConcurrentMessageLoop> submission_task_runner_;
```

Comment:
Can't we just use the worker task runner?

Reply:
I thought I had read there might be some threading restriction on which threads we can call submit from, so it needed to be the same one or something. I can't find that now, let me try this out and see

Reply:
this seems to work with just using the concurrent worker